pevm/Cargo.toml

```diff
@@ -0,0 +1,80 @@
+[package]
```

Comment:
Let's add a `crates` directory to make it more "standard", so this should be `pevm/crates/pevm/Cargo.toml`.